website/docs/index.html.markdown

```diff
@@ -137,6 +137,8 @@ For some advanced scenarios, such as where more granular permissions are necessa
 
 * `skip_credentials_validation` - (Optional) Should the AzureRM Provider skip verifying the credentials being used are valid? This can also be sourced from the `ARM_SKIP_CREDENTIALS_VALIDATION` Environment Variable. Defaults to `false`.
 
+~> **Note:** if `skip_credentials_validation` is false, AzureRM Provider will send request to list all available providers.
+
```

Comment:
```suggestion
* `skip_credentials_validation` - (Optional) Should the AzureRM Provider skip verifying the credentials being used are valid? If `false` the provider will send a request to list all available Resource Providers. This can also be sourced from the `ARM_SKIP_CREDENTIALS_VALIDATION` Environment Variable. Defaults to `false`.

```